activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   `Time#change` and methods that call it (eg. `Time#advance`) will now
+    return a `Time` with the timezone argument provided, if the caller was
+    initialized with a timezone argument.
+
+    Fixes [#42467](https://github.com/rails/rails/issues/42467).
+
+    *Alex Ghiculescu*
+
+
```

Comment:
Knit: ✂️ one too many `\n`

Reply:
We normally leave two line spaces between versions

Reply:
@pixeltrix Ahh, good to know! I didn't realize the difference between stable branch changelogs, thank you for sharing!